.eslintrc.json

```diff
@@ -276,6 +279,16 @@
     "sort-imports": "off",
     "symbol-description": "error",
     "template-curly-spacing": ["error", "never"],
-    "yield-star-spacing": ["error", "after"]
+    "yield-star-spacing": ["error", "after"],
+
+    // Custom rules
+    "jsdom-internal/hook-super-invocation": ["error",
```

Comment:
@domenic @Zirro The hooks configuration is now part of the `.eslintrc`. It contains the hooks name along with the ancestor class defining the hook. This way no need to disable the rule for the base class anymore. Let me know what you think.